data/eFLOWER_Dun_2022/metadata.yml

```diff
@@ -44,7 +49,17 @@ dataset:
     folder
   notes: none
 locations: .na
-contexts: .na
+contexts:
+- context_property: replicate observations
```

Comment:
Should this also be "repeat observations"?

Reply:
So here I feel like it is different - there isn't a time component - they are simply 2 measurements (from different sources, where we haven't mapped in source - and won't be at this point)